client/src/actions/types.js

```diff
@@ -1,16 +1,15 @@
 /**
  * This file houses our Action Creator type constants.
  */
-export const AUTH_SIGN_IN_REQUEST = 'AUTH_SIGN_IN_REQUEST';
-export const AUTH_SIGN_IN_SUCCESS = 'AUTH_SIGN_IN_SUCCESS';
-export const AUTH_SIGN_IN_FAIL = 'AUTH_SIGN_IN_FAIL';
-export const AUTH_SIGN_OUT = 'AUTH_SIGN_OUT';
+export const SIGN_IN_REQUEST = 'IGN_IN_REQUEST';
+export const SIGN_IN_SUCCESS = 'SIGN_IN_SUCCESS';
+export const SIGN_IN_FAIL = 'SIGN_IN_FAIL';
+export const SIGN_OUT = 'SIGN_OUT';
+export const REGISTRATE_SUCCESS = 'REGISTRATE_SUCCESS';
```

Comment:
Can  replace this to `REGISTER_SUCCESS`/`REGISTRATION_SUCCESS`/similar? Ditto naming convention in other files.